roxie/roxiemem/roxiemem.cpp

```diff
@@ -3415,8 +3415,8 @@ class BufferedRowCallbackManager
     Owned<BackgroundReleaseBufferThread> backgroundReleaseBuffersThread;
     Owned<ReleaseBufferThread> releaseBuffersThread;
     CRowManager * owner;
-    atomic_t releasingBuffers;  // boolean if pre-emptive releasing thread is active
-    atomic_t releaseSeq;
+    std::atomic_uint releasingBuffers;  // boolean if pre-emptive releasing thread is active
```

Comment:
I prefer always qualifying (when referring to an external namespace) but not qualifying when code within the namepsace refers to other code within the namespace